Src/Plugins/Tests.Security/EndToEndTests.cs

```diff
@@ -137,6 +138,35 @@ protected void RunAllTests()
             }
 
             RunTest(inputFiles, expectedOutputResourceMap, enforceNotificationsFree: true);
+
+            ValidateOutputFiles();
+        }
+
+        private void ValidateOutputFiles()
```

Comment:
>ValidateOutputFiles [](http://example.com/codeflow?start=21&length=19)

This doesn't look like a high value test to me.

Also, if you drop this in, we won't be able to use this mechanism to create test cases that are designed specifically to *not* fire results. 

I suggest we drop it unless you feel strongly about the value.

Reply:
just removed.